smarts/core/__init__.py

```diff
@@ -35,5 +35,4 @@ def seed(a):
 
 
 def gen_id():
-    """Generates a unique but deterministic id if `smarts.core.seed` has set the core seed."""
-    return uuid.UUID(int=random.getrandbits(128))
+    return str(uuid.uuid4())[:8]
```

Comment:
explain? I thought we used the `random.getrandbits` so that we are using the same seed to generate these UUID's